packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -56,25 +56,36 @@ const gcOptionsMatrix: OptionsMatrix<IGCRuntimeOptions> = {
     sessionExpiryTimeoutMs: [undefined], // Don't want coverage here
 };
 
-const summaryOptionsMatrix: OptionsMatrix<ISummaryRuntimeOptions> = {
-    disableSummaries: [false],
-    initialSummarizerDelayMs: numberCases,
-    summaryConfigOverrides: [undefined],
-    maxOpsSinceLastSummary: numberCases,
+const summaryConfigurationMatrix: OptionsMatrix<ISummaryConfigurationHeuristics> = {
+    state: ["enabled"],
+    minIdleTime: [0],
+    maxIdleTime: [30 * 1000], // 30 secs.
+    maxTime: [60 * 1000], // 1 min.
+    maxOps: [100], // Summarize if 100 weighted ops received since last snapshot.
+    minOpsForLastSummaryAttempt: [10],
+    maxAckWaitTime: [10 * 60 * 1000], // 10 mins.
+    maxOpsSinceLastSummary: [400, 800, 2000],
```

Comment:
maxOpsSinceLastSummary and initialSummarizerDelayMs were numberCases before. Why not keep them numberCases now?
IMO, all of these should be number cases. Maybe we put a min or max on some of them but otherwise we should be testing with all possible values.

@anthony-murphy I would love to hear your feedback on this.

Reply:
I will start with this set and will add/change new sets gradually.

Reply:
@NicholasCouri this is wrong. i assume generally clients won't set any values here, so they should all definitely include undefined, so we test the defaults.

Reply:
@anthony-murphy - the problem is that we designed it to enforce the consumers to explicitly define each value once it gets overridden.  The values I'm manually setting are the default ones + the ones I want to exercise at the moment. I will gradually change/add more variations for the values.